_posts/2019-05-02-terrence-lee-keynote.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "Keynote speaker Terence Lee"
+---
+We are thrilled to announce Terence Lee as the keynote Speaker of Ruby Unconf 2019! Terence will held his keynote about Ruby on Saturday the 25th of May.
+
+Terence leads Heroku’s Ruby Task Force curating the Ruby experience on the platform. He has worked on several open source software projects such as Ruby (itself), mruby, mruby-cli, Bundler, Resque. When he is not contributing to OSS projects he is helping with the Rails Girls movement. Terence likes to get together with people so you will probably meet him at the one or other awesome Ruby event. At least once there will be a karaoke session with him. For this occasion, he created the [#rubykaraoke](https://twitter.com/hashtag/rubykaraoke?src=hash) Twitter hashtag. When not traveling the world to attend conferences, Terence lives in Austin, TX, the taco capital of America.
+
```

Comment:
Do we have a nice picture of Terence we could add here? 📸

Reply:
Ah thanks for the reminder